polkadot/primitives/src/v6/mod.rs

```diff
@@ -1561,7 +1568,7 @@ impl parity_scale_codec::Decode for CompactStatement {
 	) -> Result<Self, parity_scale_codec::Error> {
 		let maybe_magic = <[u8; 4]>::decode(input)?;
 		if maybe_magic != BACKING_STATEMENT_MAGIC {
-			return Err(parity_scale_codec::Error::from("invalid magic string"))
+			return Err(parity_scale_codec::Error::from("invalid magic string"));
```

Comment:
Looks like your formatter is adding a bunch of unsolicited corrections... It won't pass formatting check in CI like this. Are you using the latest nightly toolchain and `cargo +nightly fmt`?